mandatory/2-function-creation.js

```diff
@@ -14,27 +18,62 @@ function isAcceptableUser(userAge, isLoggedIn) {}
   The function should return the new price to be paid (e.g. if the totalPrice is 150, a 5% discount
     is applieds and 142.5 should be returned)
 */
+var discount;
+function applyDiscount(totalPrice) {
+  if(totalPrice>200){
+    discount=10;}
+    else{ discount=5;
+
+    }
+    return  totalPrice-( totalPrice * discount / 100);
+  }
 
-function applyDiscount(totalPrice) {}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let i = 1;
+  
+  while (i % 1===0){
+    console.log(i);
+  }
+  i = i + 1;
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
-
+function buyTwoGetTheCheapestFree(price1, price2) {
+  let totalPrice = price1 + price2;
+  if (price1 < price2){
+    let discount = price1;
+  }
+  return  totalPrice - price1;
+  }
+
+  
 /*
   Complete the function to determine if it is suitable for a person to register based on their age!
   - if the person is 12 or younger it should return "You Are Too Young To Register"
   - if the person is older than 12 and younger than 90 it should return "You Can Register"
   - if the person is 90 or older it should return "You Don't Need To Register"
 */
-function canRegister(age) {}
+function canRegister(age) {
+  if (age <= 12){
+    return "You Are Too Young To Register"
+  }
+   else if (age > 12 && age < 90) {
+    return "You Can Register"
+   }
+   else {
+    return "You Don't Need To Register"
+   }
+
+  
+
+}
 
 /*
   Complete the function so that it prints out to the console numbers in reverse order starting at
```

Comment:
Looks like you have missed 1 more step of your assignment `function countReverse(number) {}`